lessons/05-html-linking/document/resume.html

```diff
@@ -0,0 +1,28 @@
+<html>
+    <head>
+        <title>
+            resume
+        </title>
+    </head>
+    <body>
+        <h1>Resume</h1>
+        <ul>
+            <li><a href="../index.html">home</a></li>
+            <li><a href="../news.html">news</a></li>
+            <li><a href="../about.html">about</a></li>
+            <li><a href="../contact.html">contact</a></li>
+        </ul>
+       <h1>Personal Info</h1><hr>
+       <br>
+       <sub><strong>Name:</strong>Noor Al-Sabbagh</sub><br>
+       <br>
+       <sub><strong>Address:</strong>123 Fake St,SK Canada</sub><br>
+       <br>
+       <sub><strong>Phone:</strong>306-123-4567</sub><br>
+       <br>
```

Comment:
Instead of separating everything with a `<br />` tag, try just putting the text itself inside of a `<p>` tag. Each `<p>` will be on its own line

```html
<p>
  <sub><strong>Name:</strong>Noor Al-Sabbagh</sub>
</p>

<p>
  <sub><strong>Address:</strong>123 Fake St,SK Canada</sub>
</p>

...etc
```